src/components/scenes/FioRequestListScene.js

```diff
@@ -0,0 +1,286 @@
+// @flow
+
+import type { EdgeDenomination } from 'edge-core-js'
+import React, { Component } from 'react'
+import { Alert, FlatList, TouchableOpacity, View } from 'react-native'
+import { Actions } from 'react-native-router-flux'
+import slowlog from 'react-native-slowlog'
+import { SwipeListView, SwipeRow } from 'react-native-swipe-list-view'
+
+import * as Constants from '../../constants/indexConstants'
+import { intl } from '../../locales/intl'
+import s from '../../locales/strings.js'
+import type { ExchangeRatesState } from '../../modules/ExchangeRates/reducer'
+import FioRequestRow from '../../modules/FioRequest/components/FioRequestRow'
+import T from '../../modules/UI/components/FormattedText/index'
+import { styles as requestListStyles } from '../../styles/scenes/FioRequestListStyle'
+import styles from '../../styles/scenes/TransactionListStyle'
+import type { FioRequest, GuiWallet } from '../../types/types'
+import FullScreenLoader from '../common/FullScreenLoader'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { showError } from '../services/AirshipInstance'
+
+const SCROLL_THRESHOLD = 0.5
+
+export type State = {
+  rejectLoading: boolean
+}
+
+export type StateProps = {
+  wallets: { [walletId: string]: GuiWallet },
+  loading: boolean,
+  isoFiatCurrencyCode: string,
+  fiatSymbol: string,
+  displayDenominations: { [string]: EdgeDenomination },
+  exchangeRates: ExchangeRatesState,
+  fioRequestsPending: FioRequest[],
```

Comment:
fioRequestsPending and fioRequestsSent need not be in redux. Refactor to make it local component state and rebase this commit. Dont push a new commit.

Reply:
not sure we should move this to the component. maybe in future there would be some functionality that can use this. also component would be too big with all of the getting requests functionality and reject

Reply:
done